scripts/initialize.sh

```diff
@@ -107,18 +111,43 @@ if [ ! -z "$authorization_amount" ]; then
    authorization_amount_opt="--authorization ${authorization_amount}"
 fi
 
-cd $KEEP_BEACON_SOL_PATH
-    
 printf "${LOG_START}Setting up staking...${LOG_END}"
 
-stake="npx hardhat initialize \
-    --network $NETWORK \
-    --owner ${stake_owner} \
-    --provider ${staking_provider} \
-    --operator ${operator} \
-    --beneficiary ${beneficiary} \
-    --authorizer ${authorizer}"
+mint="npx hardhat initialize:mint 
+   --network $NETWORK \
+   --owner ${stake_owner}"
+stake="npx hardhat initialize:stake 
+   --network $NETWORK \
+   --owner ${stake_owner} \
+   --provider ${staking_provider} \
+   --beneficiary ${beneficiary} \
+   --authorizer ${authorizer}"
+authorize="npx hardhat initialize:authorize 
+   --network $NETWORK \
+   --owner ${stake_owner} \
+   --provider ${staking_provider} 
+   --authorizer ${authorizer}"
+register="npx hardhat initialize:register 
+   --network $NETWORK \
+   --provider ${staking_provider} \
+   --operator ${operator}"
```

Comment:
With changes from https://github.com/keep-network/keep-core/pull/3123 we could call just the `initialize` task, no need to call all subtasks separately.